### PR TITLE
Update readme to reflect the state of this project

### PR DESCRIPTION
**Pull request by MiniDigger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 Minetrack is a Minecraft PC/PE server tracker that lets you focus on what's happening *now*. 
 Built to be lightweight and durable, you can easily adapt it to monitor BungeeCord or server instances.
 
+#### A note on the state of this project
+This project and the offical website are not actively maintained anymore, but you are welcome to run your once instances of minetrack.
+
```
Comment (MiniDigger) [inline README.md]:
```suggestion
I will however accept and review pull-requests, so please share any improvements you are making so everybody can benefit from them.
```

Comment (MiniDigger) [inline README.md]:
```suggestion

You can find a list of community hosted instances below:
* https://minetrack.me

Want to be listed here? Add yourself in a pull-request!
```

**Comment by MiniDigger** [comment]
the two suggestions are just ideas, alternativly you could archive this project.

File: README.md (hunk 1 above)
Comment (Cryptkeeper) [reply README.md]: Looks like there's a typo of once that should be "own", and Minetrack instead of minetrack.

Comment (Cryptkeeper) [reply README.md]: Github won't let me add this suggestion, could you add it in yourself and push?

**Comment by Cryptkeeper** [comment]
A few quick fixes and then I'm happy to merge it into master. Thanks!

**Comment by MiniDigger** [comment]
done, please use the "squash and merge" feature so I dont have to checkout the project and squash manually ^^

**Pull request merged**